lib/auth/touchid/api_darwin.go

```diff
@@ -50,6 +51,10 @@ const (
 	// rpID are domain names, so it's safe to assume they won't have spaces in them.
 	// https://www.w3.org/TR/webauthn-2/#relying-party-identifier
 	labelSeparator = " "
+
+	// promptReason is the LAContext / Touch ID prompt.
+	// The final prompt is: "$binary is trying to authenticate user".
+	promptReason = "authenticate user"
```

Comment:
nit: "authenticate the user" perhaps?

Reply:
I copied the Apple default prompt, which says exactly that in my machine. (YMMV due to localization.)